java-engine/src/main/java/io/getunleash/engine/UnleashEngine.java

```diff
@@ -5,6 +5,7 @@
 import com.fasterxml.jackson.databind.DeserializationFeature;
 import com.fasterxml.jackson.databind.ObjectMapper;
 import com.fasterxml.jackson.datatype.jsr310.JavaTimeModule;
+import com.sun.jna.Memory;
```

Comment:
I just spotted this and I believe com.sun.jna is only part of Oracle's distribution of Java. It can be added as a dependency alongside Unleash but I'm not sure if we want to go this route.

Reply:
It comes from this library: https://github.com/java-native-access/jna, which is LGPL licensed and we've been testing this on OpenJDK, I think it's fine. This is already an import, it's just new in this file

We may yeet JNA for different reasons though